picasso/src/main/java/com/squareup/picasso/Picasso.java

```diff
@@ -768,14 +768,22 @@ public Builder executor(@NonNull ExecutorService executorService) {
     }
 
     /** Specify the memory cache used for the most recent images. */
-    public Builder memoryCache(@NonNull Cache memoryCache) {
-      if (memoryCache == null) {
-        throw new IllegalArgumentException("Memory cache must not be null.");
+    public Builder memoryCacheBytes(int maxByteCount) {
```

Comment:
`withCacheSize(int)`?

Reply:
yes. dig it. and then 0 is no cache. thanks.